kore/test/Test/Kore/Rewrite/MockSymbols.hs

```diff
@@ -46,6 +46,7 @@ import Kore.Attribute.Pattern.ConstructorLike (
     isConstructorLike,
  )
 import Kore.Attribute.Sort qualified as Attribute
+import Kore.Attribute.Sort qualified as AttributeSort
```

Comment:
Curious... why do we keep the line above importing as `Attribute`? Maybe we could drop that one?

Reply:
This whole thing is a mess. I don't know the whole story, but apparently `DuplicateRecordFields` is changing (or going away?) and we have to deal with that. I have never really used that extension, and have no idea what the right approach may be. I can try to find out what's changing. As for that specific thing, yes, I can just use one, but IIRC a lot more lines will have to change.

Reply:
OK, so better leave it as-is then.
I never mastered `DuplicateRecordFields`  and its extension friends for better records - but we use `OverloadedRecordDot`  in another project, which IMHO is real progress compared to before (and I was working in the shop that was driving that proposal so I am very used to writing code with that extension).

Reply:
OK, so it's changing, not going away, but losing its type-directed disambiguation. I have no idea why the access functions for it weren't just changed to behave like `OverloadedRecordDot`, but whatever. In the access cases, we *could* switch to dots, but that won't do anything for the update cases. Leaving it as is seems okay; if someone wants to do something more stylish later that's fine too.